src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -889,6 +889,18 @@
     crafted: true
 """
 
+helps['vm boot-diagnostics get-sas'] = """
```

Comment:
sas is [shared access signature](https://docs.microsoft.com/en-us/samples/azure-samples/storage-dotnet-sas-getting-started/storage-dotnet-sas-getting-started/), while the [request](https://github.com/Azure/azure-cli/issues/14539) is asking to get boot log url. Suggest to rename to `get-log`

Reply:
A sample response:
```
{
  "consoleScreenshotBlobUri": "https://md-visib0spjzkmsn3z.blob.core.windows.net:443/3289a9bc-c41f-4750-9595-a6a41500fc30/vm1.66212157-7e08-48e9-bb0b-1b31fad6ea79.screenshot.bmp?sv=2017-04-17&sr=b&sig=z5YDfqrK93Lhd8QzWzTeRquDFVLalKmbkt5nZWppIk8%3d&se=2020-09-14T08%3a03%3a52Z&sp=r",
  "serialConsoleLogBlobUri": "https://md-visib0spjzkmsn3z.blob.core.windows.net:443/3289a9bc-c41f-4750-9595-a6a41500fc30/vm1.66212157-7e08-48e9-bb0b-1b31fad6ea79.serialconsole.log?sv=2017-04-17&sr=b&sig=aaQNEtL2D%2bBgT1VtLjFkMx%2buaNRrMxM3CrTcXxwyvUs%3d&se=2020-09-14T08%3a03%3a52Z&sp=r"
}
```
how about get-blog-uri?

Reply:
thanks @qwordy . As we synced offline, with new managed storage  for boot diagnostic log,  2 changes need.
-  **MUST** change existing get_boot_log command to support showing content from managed storage uri, when log is managed storage, get uri from the new InstanceView API
- **Nice to have** add a new command `get_boot_log_uris` to show boot log uris. This new command need to cover both scenarios also: managed storage, and user specified storage.

Reply:
Done